cmd/ccboc/main.go

```diff
@@ -353,10 +357,12 @@ func main() {
 						Usage:   "get results of calculations",
 						Action: func(c *cli.Context) error {
 							var err error
-							if c.Float64("teff") == 0 || c.Float64("logG") == 0 {
+							if c.Float64("teff") == 0 && c.Float64("logG") == 0 {
```

Comment:
We need to validate if any of teff or logG are empty and return an error instead.

Reply:
Aren't they 0 by default? I can then split these 2 commands just like get calculation/calculations. Is that better?

Reply:
Right now, when teff and logG have other value than 0 (both of them) we call `getResult()`.
If both of them are empty = 0, we call `getResultById()`, where we then check if the `calcID` was specified.
Else, we return an error message.